.github/workflows/openw3d-linux.yml

```diff
@@ -0,0 +1,29 @@
+name: openw3d-linux
```

Comment:
Untested, but I think we can and should merge all workflows.
It will spare us lots of headaches in the futures.

The following patch should add Linux to the already existing one (untested):
```patch
--- a/.github/workflows/openw3d-windows.yml
+++ b/.github/workflows/openw3d-windows.yml
@@ -42,6 +42,9 @@ jobs:
             msys2-msystem: mingw64
             msys2-env: mingw-w64-x86_64
             shell: msys2 {0}
+          - name: Linux x86_64
+            os: ubuntu-latest
+            build-args: --parallel --target scripts
 
     steps:
 
```
(`.github/workflows/openw3d-windows.yml` should then obviously be renamed)

Reply:
I don't have strong feelings one way or the other, but the reason I've done separate workflows in the past is because windows tends to be _just_ different enough from Linux that it becomes a big file with a bunch of special cases depending on if you're compiling on windows or linux.

Reply:
To give a single data point, until last year libsdl-org/SDL had a separate workflow for every platform: windows, linux/macos, ps2, psp, vita, haiku, freebsd, netbsd, watcom, tvos/ios, n3ds, riscos, emscripten, android. 14+ workflows.
Reading the post-report and/or canceling an erroneous job is much easier when they are combined.
Also, [every job takes up a line](https://github.com/w3dhub/OpenW3D/actions): there is an entry for openw3d-windows and openw3d-linux.